plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -35,10 +35,23 @@ import (
 )
 
 const (
+	bucketRange        = `(0,1,2,3,4,5,6,7,8,9)`
```

Comment:
i'd like to punt on this ask - it's going to be a hassle to verify the behavior of ? in an IN clause.